hearth/sig/lib/hearth/interfaces.rbs

```diff
@@ -10,52 +10,53 @@ end
 class Logger
 end
 
-interface _Plugin[T]
-  def call: (Hearth::Configuration[T]) -> void
-end
+module Hearth
+  interface _Plugin
+    def call: (Configuration) -> void
```

Comment:
Ditto here - this takes a service's Config - so I think it should be paramterized.